modules/machine-autoscaler-crd.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * control-plane-management/applying-autoscaling.adoc
+
+[id='machine-autoscaler-crd-{context}']
+= MachineAutoscaler resource definition
+
+This MachineAutoscaler resource definition shows the parameters and sample
+values for the MachineAutoscaler.
+
+
+[source,yaml]
+----
+apiVersion: "autoscaling.openshift.io/v1alpha1"
+kind: "MachineAutoscaler"
+metadata:
+  name: "worker-us-east-1a" <1>
+  namespace: "openshift-cluster-api"
+spec:
+  minReplicas: 1 <2>
+  maxReplicas: 12 <3>
+  scaleTargetRef:
+    apiVersion: machine.openshift.io/v1alpha1
+    kind: MachineSet
+    name: worker <4>
+----
+<1> Specify the `MachineAutoscaler` name, which is composed of the name of the
+MachineSet to scale and the Amazon Web Services (AWS) zone to scale in.
+The value uses the following format:
+`"<MachineSet_name>-<Amazon_Web_Services-zone>"`.
+<2> Specify the minimum number Machines of the specified type to deploy in the
+specified AWS zone.
+<3> Specify the maxiumum number Machines of the specified type to deploy in the
+specified AWS zone.
+<4> Specify the type of Machine to scale.
```

Comment:
I would maybe reword this a bit. The `scaleTargetRef` is a reference by name to an existing MachineSet you want to scale. The `apiVersion` and `kind` will almost always be what's in the example for OpenShift users, and the name will be the name of an already existing MachineSet in the cluster.

Reply:
@bison, is there a good way to get the correct  `apiVersion` and `kind` from the CLI, or is it best to go digging through the UI?

Reply:
Hmmm, well it's whatever the type of the MachineSet you want to scale is, I guess you could describe the existing MachineSet to see it: `oc -n openshift-machine-api describe $name`

For the foreseeable future in OpenShift, it's always going to be `apiVersion: machine.openshift.io/v1beta1` and `kind: MachineSet` though.

Reply:
Thank you so much! If you remember, I'd love a nudge when the `apiVersion` changes so I can update the sample.

Reply:
@kalexand-rh, I think our example is updated now:
https://github.com/openshift/cluster-autoscaler-operator/blob/master/examples/machineautoscaler.yaml